server/users.ts

```diff
@@ -598,13 +598,11 @@ class User extends Chat.MessageContext {
 	}
 	getIdentityWithStatus(roomid: string = '') {
 		const identity = this.getIdentity(roomid);
-		const status = this.statusType === 'online' ? '' : '@!';
+		const status = this.getStatus();
 		return `${identity}${status}`;
 	}
 	getStatus() {
-		const statusMessage = this.statusType === 'busy' ? '!(Busy) ' : this.statusType === 'idle' ? '!(Idle) ' : '';
-		const status = statusMessage + (this.userMessage || '');
-		return status;
+		return this.statusType === 'idle' ? '@!' : this.statusType === 'busy' ? '@' : /* online */ '';
```

Comment:
Whether Idle should be `@` or `@?` is up on the fence still, I think?
```suggestion
		return this.statusType === 'busy' ? '@!' : this.statusType === 'idle' ? '@' : /* online */ '';
```

Reply:
No, Zarel is pretty vehement about Busy as `@` and Idle as `@!`, so the code as written should correctly reflect his preference.

Reply:
Oh, I reverted it in my brain and thought he wanted `@!` for busy and `@` idle, my bad.

Reply:
Yeah, *we* want `@!` for Busy which is why its so confusing, but to Zarel, Busy is ⊥, and I'm tired to arguing.